shared/scss/_vars.scss

```diff
@@ -105,4 +105,3 @@ $icon__settings--moz: "moz-extension://__MSG_@@extension_id__/img/settings-gear@
 /* Close modal "x" icon */
 $icon__close--chrome: "chrome-extension://__MSG_@@extension_id__/img/close.svg";
 $icon__close--moz: "moz-extension://__MSG_@@extension_id__/img/close.svg";
```

Comment:
Can you fix your code editor so that it does not remove the blank line at the end of files? Thanks. Details are in the "Code editor setup" task in the "O-1 Meta" project.

Reply:
Ah, I did this manually when I removed some variables from here. I'll readd.